app/config/messaging.js

```diff
@@ -1,5 +1,4 @@
 const sharedConfig = {
-  appInsights: require('applicationinsights'),
```

Comment:
We need this as it enables app insights inside the ffc-messaging module

Reply:
Done.